CRITICAL - 2022-04-08 14:18:18 --> Arquivo invÃ¡lido: testeci4.php
#0 C:\Users\Adasi\Documents\Project_food\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile()
#1 C:\Users\Adasi\Documents\Project_food\system\Common.php(1089): CodeIgniter\View\View->render()
#2 C:\Users\Adasi\Documents\Project_food\app\Controllers\Home.php(9): view()
#3 C:\Users\Adasi\Documents\Project_food\system\CodeIgniter.php(825): App\Controllers\Home->index()
#4 C:\Users\Adasi\Documents\Project_food\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#5 C:\Users\Adasi\Documents\Project_food\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#6 C:\Users\Adasi\Documents\Project_food\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 C:\Users\Adasi\Documents\Project_food\system\Commands\Server\rewrite.php(43): require_once('C:\\Users\\Adasi\\...')
#8 {main}
ERROR - 2022-04-08 15:24:58 --> ErrorException: pg_query(): Query failed: ERROR:  relation "expediente" does not exist
LINE 1: INSERT INTO "expediente" ("abertura", "fechamento", "situaca...
                    ^ in C:\Users\Adasi\Documents\Project_food\system\Database\Postgre\Connection.php:135
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler()
#1 C:\Users\Adasi\Documents\Project_food\system\Database\Postgre\Connection.php(135): pg_query()
#2 C:\Users\Adasi\Documents\Project_food\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute()
#3 C:\Users\Adasi\Documents\Project_food\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery()
#4 C:\Users\Adasi\Documents\Project_food\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query()
#5 C:\Users\Adasi\Documents\Project_food\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\Users\Adasi\Documents\Project_food\system\BaseModel.php(738): CodeIgniter\Model->doInsert()
#7 C:\Users\Adasi\Documents\Project_food\system\Model.php(625): CodeIgniter\BaseModel->insert()
#8 C:\Users\Adasi\Documents\Project_food\app\Database\Seeds\ExpedienteSeeder.php(69): CodeIgniter\Model->insert()
#9 C:\Users\Adasi\Documents\Project_food\system\Database\Seeder.php(147): App\Database\Seeds\ExpedienteSeeder->run()
#10 C:\Users\Adasi\Documents\Project_food\system\Commands\Database\Seed.php(77): CodeIgniter\Database\Seeder->call()
#11 C:\Users\Adasi\Documents\Project_food\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\Seed->run()
#12 C:\Users\Adasi\Documents\Project_food\system\CLI\CommandRunner.php(70): CodeIgniter\CLI\Commands->run()
#13 C:\Users\Adasi\Documents\Project_food\system\CLI\CommandRunner.php(56): CodeIgniter\CLI\CommandRunner->index()
#14 C:\Users\Adasi\Documents\Project_food\system\CodeIgniter.php(823): CodeIgniter\CLI\CommandRunner->_remap()
#15 C:\Users\Adasi\Documents\Project_food\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#16 C:\Users\Adasi\Documents\Project_food\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#17 C:\Users\Adasi\Documents\Project_food\system\CLI\Console.php(48): CodeIgniter\CodeIgniter->run()
#18 C:\Users\Adasi\Documents\Project_food\spark(63): CodeIgniter\CLI\Console->run()
#19 {main}
ERROR - 2022-04-08 15:26:12 --> ErrorException: pg_query(): Query failed: ERROR:  relation "expediente" does not exist
LINE 1: INSERT INTO "expediente" ("abertura", "fechamento", "situaca...
                    ^ in C:\Users\Adasi\Documents\Project_food\system\Database\Postgre\Connection.php:135
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler()
#1 C:\Users\Adasi\Documents\Project_food\system\Database\Postgre\Connection.php(135): pg_query()
#2 C:\Users\Adasi\Documents\Project_food\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute()
#3 C:\Users\Adasi\Documents\Project_food\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery()
#4 C:\Users\Adasi\Documents\Project_food\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query()
#5 C:\Users\Adasi\Documents\Project_food\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\Users\Adasi\Documents\Project_food\system\BaseModel.php(738): CodeIgniter\Model->doInsert()
#7 C:\Users\Adasi\Documents\Project_food\system\Model.php(625): CodeIgniter\BaseModel->insert()
#8 C:\Users\Adasi\Documents\Project_food\app\Database\Seeds\ExpedienteSeeder.php(69): CodeIgniter\Model->insert()
#9 C:\Users\Adasi\Documents\Project_food\system\Database\Seeder.php(147): App\Database\Seeds\ExpedienteSeeder->run()
#10 C:\Users\Adasi\Documents\Project_food\system\Commands\Database\Seed.php(77): CodeIgniter\Database\Seeder->call()
#11 C:\Users\Adasi\Documents\Project_food\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\Seed->run()
#12 C:\Users\Adasi\Documents\Project_food\system\CLI\CommandRunner.php(70): CodeIgniter\CLI\Commands->run()
#13 C:\Users\Adasi\Documents\Project_food\system\CLI\CommandRunner.php(56): CodeIgniter\CLI\CommandRunner->index()
#14 C:\Users\Adasi\Documents\Project_food\system\CodeIgniter.php(823): CodeIgniter\CLI\CommandRunner->_remap()
#15 C:\Users\Adasi\Documents\Project_food\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#16 C:\Users\Adasi\Documents\Project_food\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#17 C:\Users\Adasi\Documents\Project_food\system\CLI\Console.php(48): CodeIgniter\CodeIgniter->run()
#18 C:\Users\Adasi\Documents\Project_food\spark(63): CodeIgniter\CLI\Console->run()
#19 {main}
ERROR - 2022-04-08 15:27:15 --> ErrorException: pg_query(): Query failed: ERROR:  relation "expediente" does not exist
LINE 1: INSERT INTO "expediente" ("dia", "dia_descricao", "abertura"...
                    ^ in C:\Users\Adasi\Documents\Project_food\system\Database\Postgre\Connection.php:135
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler()
#1 C:\Users\Adasi\Documents\Project_food\system\Database\Postgre\Connection.php(135): pg_query()
#2 C:\Users\Adasi\Documents\Project_food\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute()
#3 C:\Users\Adasi\Documents\Project_food\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery()
#4 C:\Users\Adasi\Documents\Project_food\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query()
#5 C:\Users\Adasi\Documents\Project_food\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\Users\Adasi\Documents\Project_food\system\BaseModel.php(738): CodeIgniter\Model->doInsert()
#7 C:\Users\Adasi\Documents\Project_food\system\Model.php(625): CodeIgniter\BaseModel->insert()
#8 C:\Users\Adasi\Documents\Project_food\app\Database\Seeds\ExpedienteSeeder.php(74): CodeIgniter\Model->insert()
#9 C:\Users\Adasi\Documents\Project_food\system\Database\Seeder.php(147): App\Database\Seeds\ExpedienteSeeder->run()
#10 C:\Users\Adasi\Documents\Project_food\system\Commands\Database\Seed.php(77): CodeIgniter\Database\Seeder->call()
#11 C:\Users\Adasi\Documents\Project_food\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\Seed->run()
#12 C:\Users\Adasi\Documents\Project_food\system\CLI\CommandRunner.php(70): CodeIgniter\CLI\Commands->run()
#13 C:\Users\Adasi\Documents\Project_food\system\CLI\CommandRunner.php(56): CodeIgniter\CLI\CommandRunner->index()
#14 C:\Users\Adasi\Documents\Project_food\system\CodeIgniter.php(823): CodeIgniter\CLI\CommandRunner->_remap()
#15 C:\Users\Adasi\Documents\Project_food\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#16 C:\Users\Adasi\Documents\Project_food\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#17 C:\Users\Adasi\Documents\Project_food\system\CLI\Console.php(48): CodeIgniter\CodeIgniter->run()
#18 C:\Users\Adasi\Documents\Project_food\spark(63): CodeIgniter\CLI\Console->run()
#19 {main}
